metadata-ingestion/src/datahub/ingestion/source/sql/sql_types.py

```diff
@@ -468,7 +468,7 @@ def resolve_vertica_modified_type(type_string: str) -> Any:
 
 
 def resolve_sql_type(
-    column_type: Optional[str],
+    column_type: Optional[str] = None,
```

Comment:
I don't think this change makes sense

Reply:
yeah let's change it back - the column_type should be a required param. The only reason None is allowed is to make it easy to handle cases where we were unable to fetch type info from the underlying source